common/ajax-fetch/ajax-fetch-en.html

```diff
@@ -0,0 +1,45 @@
+---
+{
+	"title": "Ajax Fetch",
+	"language": "en",
+	"altLangPage": "ajax-fetch-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-25"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<h2>Successful Ajax Fetch</h2>
+<div data-ajax-replace="https://wet-boew.github.io/wet-boew/demos/data-ajax/ajax/data-ajax-extra-en.html">
```

Comment:
There is an issue with the content that are inserted. We end up with skipping the heading level 3.

See:
![image](https://github.com/wet-boew/GCWeb/assets/1676030/ec3e38d4-5797-4e36-93f3-fb10598bea74)


I suggest to fix this issue to add and adapt the HTML fragment file for this example.

You can copy the [content from the data-ajax](https://github.com/wet-boew/wet-boew/blob/master/src/plugins/data-ajax/ajax/data-ajax-extra-en.html) and move it here in a new folder named "ajax" + change the heading for an h3

```suggestion
<div data-ajax-replace="ajax/data-ajax-extra-en.html">
```